src/test/sanity_tests.cpp

```diff
@@ -1,19 +0,0 @@
-// Copyright (c) 2012-2021 The Bitcoin Core developers
-// Distributed under the MIT software license, see the accompanying
-// file COPYING or http://www.opensource.org/licenses/mit-license.php.
-
-#include <key.h>
-#include <test/util/setup_common.h>
-#include <util/time.h>
-
-#include <boost/test/unit_test.hpp>
-
-BOOST_FIXTURE_TEST_SUITE(sanity_tests, BasicTestingSetup)
```

Comment:
In commit "test: Run AppInitSanityChecks before all tests" (facdfb61b8df2958e47ced2c5014383fcf5affef)

I wish this commit would just add the AppInitSanityChecks assert without deleting the other tests. I don't see a reason to delete low level tests just because high level code calls some of the same functions. Low level tests can be useful because they can find identify errors that might be masked in high level tests that are changing other state. Also they can be provide more granular errors in case of failure, and they can serve as examples to show how low-level APIs can be called.